ccgains/trades.py

```diff
@@ -1041,6 +1091,10 @@ def append_electrum_csv(self, file_name, skiprows=1,
 
         Afterwards, all trades will be sorted by date and time.
 
+        :param add_missing_fees: bool
+            Indicates whether the missing fees should be obtained using the
+            'blockcypher' API.
```

Comment:
Here also, please add a warning, same as above, something like: "Please note that using this introduces a privacy leak: Since 'blockcypher' will be asked about all your withdrawals, they will be able to exactly tell which wallets belong to your IP."